pygenie/jobs/utils.py

```diff
@@ -133,9 +134,9 @@ def wrapper(*args, **kwargs):
 
         final = list()
         last = None
-        for item in sorted(attr):
+        for item in sorted([json.dumps(a) for a in attr]):
```

Comment:
Does this work in Python 3?  Doesn't dumps cause these to become typed so that sorting will fail?

Reply:
I tested this in Python 3 and it works.